yalelibrary/yul-solr
# Add new highlight fields to upgrade config

### Testing
1. Merge the branch
2. Pull changes to search-upgrade
3. Reset Solr if necessary
4. You should be able to search for the new highlighted fields
5. Search for `10 indian boys` and verify that alternative title or a subject field displays
